src/reducers/pause.js

```diff
@@ -287,25 +366,42 @@ function getPauseLocation(state, action) {
 // (right now) to type those wrapped functions.
 type OuterState = State;
 
-const getPauseState = state => state.pause;
+function getCurrentPauseState(state: OuterState): ThreadPauseState {
+  return (
+    state.pause.threads[state.pause.currentThread] || createInitialPauseState()
```

Comment:
do we need this? `createInitialPauseState()`

maybe we create this state when we see the new worker?

Reply:
This could be removed but we would need to make sure that the pause state includes entries for all threads.  Right now they are lazily initialized.